libraries/lib-track/Track.cpp

```diff
@@ -450,6 +450,28 @@ TrackList::~TrackList()
    Clear(false);
 }
 
+wxString TrackList::MakeUniqueTrackName(const wxString& baseTrackName) const
+{
+   auto name = baseTrackName;
```

Comment:
Unnecessary assignment

Reply:
You review yourself?  :-)

Reply:
Yes, I need to improve my practice of self-reviewing)